src/components/ExperimentDetail.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react'
+// import { Link } from 'react-router-dom'
+import { Container, Row, Col, Alert, Card, Progress, CardBody, CardTitle, CardSubtitle, CardText, FormGroup, Label, Input, CustomInput, ButtonDropdown, DropdownToggle, DropdownMenu, DropdownItem } from "reactstrap";
+import "../static/css/Experiments.css";
+ 
+
+const ExperimentDetail = () => {
+
+    const [data] = useState([
```

Comment:
We can actually set `data` directly instead of using `useState`.

Reply:
I.e. 
```
const data = [{ id: "1", emojis: "🚀💸", name:"Up Only WBTC" , contract: "OxSalo12345", assetPrice: "1.00", currency: "$", totalAssets: "1000", totalAUM: "10000", baseAsset: "WBTC"}]
```

Reply:
True that, I was overcomplicating, already made the change